test/conformance/testdata/conformance.yaml

```diff
@@ -1,6 +1,19 @@
+- testname: Kubelet, managed etc hosts
+  codename: '[k8s.io] KubeletManagedEtcHosts should test kubelet managed /etc/hosts
+    file [LinuxOnly] [NodeConformance] [Conformance]'
+  description: Create a Pod with containers with hostNetwork set to false, one of
+    the containers mounts the /etc/hosts file form the host. Create a second Pod with
+    hostNetwork set to true. 1. The Pod with hostNetwork=false MUST have /etc/hosts
+    of containers managed by the Kubelet. 2. The Pod with hostNetwork=false but the
+    container mounts /etc/hosts file from the host. The /etc/hosts file MUST not be
+    managed by the Kubelet. 3. The Pod with hostNetwork=true , /etc/hosts file MUST
+    not be managed by the Kubelet. This test is marked LinuxOnly since Windows cannot
+    mount individual files in Containers.
+  release: v1.9
+  file: test/e2e/common/kubelet_etc_hosts.go
 - testname: Pod Lifecycle, post start exec hook
-  codename: '[k8s.io] Container Lifecycle Hook when create a pod with lifecycle hook
-    should execute poststart exec hook properly [NodeConformance] [Conformance]'
+  codename: '[k8s.io] [sig-node] Container Lifecycle Hook when create a pod with lifecycle
```

Comment:
is the `[k8s.io]` prefix useful once these have a sig-specific prefix?

should we add this to `test/e2e/framework/framework.go`:
```golang
// SIGDescribe is a wrapper function for ginkgo describe.  Adds SIG namespacing.
func SIGDescribe(sig, text string, body func()) bool {
	return ginkgo.Describe("[sig-"+sig+"] "+text, body)
}
```

and change these calls to `SIGDescribe("node", ..., ...)`?

Reply:
At one point in time I thought `k8s.io` was used by downstream folks brave enough to build on top of the e2e binary to allow for exclusion of upstream tests.  Glancing at rows in https://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-default though it's clear this hasn't consistently applied.  So yeah, I think we should drop `k8s.io`

The pattern I'm more used to seeing is each sig-foo putting their tests in `test/e2e/foo`, which has a `SIGDescribe` function that auto-applies `[sig-foo]`.  It's a bit more boilerplate per package, but this encourages sticking to package level granularity, which is useful since OWNERS doesn't support per-file granularity.

We haven't consistently enforced this, so I do see some stray `ginkgo.Describe` calls, but at first glance, most seem redundant vs. using `SIGDescribe`.  Do we need to support per-sig tagging within a package?